docs/client_guide.md

```diff
@@ -242,6 +242,7 @@ Notice: this feature requires the MEMORY_LOCAL_HADOOP mode.
 |--------------------------------|---------|-------------------------------------------------------------------------|
 | tez.rss.avoid.recompute.succeeded.task | false   | Whether to avoid recompute succeeded task when node is unhealthy or black-listed |
 | tez.rss.client.max.buffer.size | 3k | The max buffer size in map side |
+| tez.rss.client.batch.trigger.num | 50 | The max batch of buffers to send data in map side |
```

Comment:
Could you give more detailed description? What will it happen if the task reach the value?